### PR TITLE
Updated docker command for sandbox node.

### PR DESCRIPTION
**Pull request by stevenplatt**
Updated the docker run command to include a required keypass value and docker image. 
Resolves [#9840. ](https://github.com/MinaProtocol/mina/issues/9840)

### CODE REVIEW

File: pages/en/developers/sandbox-node.mdx [hunk 1]
```diff
@@ -24,7 +24,8 @@ docker run \
   -d \
   --name mina \
   -e RUN_DEMO=true \
-  minaprotocol/mina-daemon:1.2.0-fe51f1e-devnet
+  -e CODA_PRIVKEY_PASS='' \
```
Comment (QuiteStochastic) [inline pages/en/developers/sandbox-node.mdx:27]: i think this should be `MINA_PRIVKEY_PASS` (CODA is deprecated)

File: pages/en/developers/sandbox-node.mdx [hunk 2]
```diff
@@ -24,7 +24,8 @@ docker run \
...
+  minaprotocol/mina-daemon:1.2.0-fe51f1e-demo
```
Comment (QuiteStochastic) [inline pages/en/developers/sandbox-node.mdx:28]: i wanna say that the latest version is 1.2.2, `minaprotocol/mina-daemon:1.2.2-feee67c-devnet`.  And do we even have a `-demo` container? why can't we just use the normal container?  would overall be better if we collapse these functionalities so we don't have multiple slightly different containers floating around.

File: pages/en/developers/sandbox-node.mdx (hunk 1 above)
Comment (stevenplatt) [reply pages/en/developers/sandbox-node.mdx:27]: You are correct, I will update it to "MINA_PRIVKEY_PASS". I have tested that using the mina naming works.

Comment (stevenplatt) [reply pages/en/developers/sandbox-node.mdx]: Added a new commit to update as requested.

File: pages/en/developers/sandbox-node.mdx (hunk 2 above)
Comment (stevenplatt) [reply pages/en/developers/sandbox-node.mdx]: @lk86 also agrees on the container image - I have tested ad command and updated the code for this change as well.

**Review by QuiteStochastic — APPROVED** [review]

**Pull request merged**